taipy/core/data/__init__.py

```diff
@@ -8,6 +8,7 @@
 # Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
 # an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
 # specific language governing permissions and limitations under the License.
+"""Predefined Taipy data node implementations."""
```

Comment:
And DataNode itself right?

"All the classes and functions related to Taipy data nodes."